nulib/meadow
# Delete work bug

# Summary 
Fix bug where deleting a Work throws a 404 in the app. 

# Specific Changes in this PR
- After deleting a Work, user is redirected to Search page

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
- Add a new Work from the homepage, called "Delete Me" or something you'll remember
- Go to the Preservation tab on the Work, and delete the Work.
- Notice no 404 occurs, and you're directed to the Search page.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
